allennlp/tests/commands/docstring_help_test.py

```diff
@@ -29,16 +41,8 @@ def test_docstring_help(self):
                 )
 
                 str_call_subcommand_help = match.group(1)
-                str_call_from_python_subcommand_help = str_call_subcommand_help.replace(
-                    "allennlp", "python -m allennlp.run"
-                )
-
-                actual_output = subprocess.run(
-                    str_call_from_python_subcommand_help,
-                    check=True,
-                    shell=True,
-                    stdout=subprocess.PIPE,
-                ).stdout.decode()
+                subcommand = match.group(2)
+                actual_output = _subcommand_help_output(subcommand)
 
                 pytorch_pretrained_bert_warning = (
```

Comment:
is this whole pytorch_pretrained_bert_warning part still necessary after these changes?

Reply:
No, it's not. I'm gonna remove it.